src/js/_store/modules/categories.js

```diff
@@ -0,0 +1,42 @@
+import axios from '@Helpers/axiosDefault';
+
+export default {
+  namespaced: true,
+  state: {
+    loading: false,
+    errorMessage: '',
+    doneMessage: '',
+    category: {
+      id: null,
+      name: '',
+    },
+    roleList: [],
+    categories: [],
+  },
+  mutations: {
+    doneGetAllCategories(state, payload) {
+      state.categories = [...payload.categories];
+      state.loading = false;
+    },
+    failRequest(state, { message }) {
+      state.errorMessage = message;
+      state.loading = false;
+    },
+  },
+  actions: {
+    // カテゴリー全件取得
+    getAllCategories({ commit, rootGetters }) {
+      axios(rootGetters['auth/token'])({
+        method: 'GET',
+        url: '/category',
+      }).then(response => {
+        const payload = {
+          categories: response.data.categories.reverse(),
```

Comment:
[nits]

基本的に actions は外部とのやりとりなどが責務になる（非同期関数を指定する必要があるため）ので、データの整形（今回であれば reverse()）は mutations で実装した方が良いかなと思います！

```suggestion
          categories: response.data.categories,
```

Reply:
4789ba68c8934698be66671268a4cd21d908b3b0

mutationsでreverse()を使用してカテゴリーを降順に並べるようにしました！